.github/workflows/e2e-tests.yaml

```diff
@@ -0,0 +1,84 @@
+name: Lint & E2E Checks
+on:
+  workflow_dispatch:
+  issue_comment:
+    types: [created]
+
+env:
+  docker_registry: ghcr.io/joshzcold
+  game_store: memory
+
+jobs:
+  check-comment:
+    name: Check for Maintainer Comment
+    runs-on: ubuntu-latest
+    outputs:
+      should_run: ${{ steps.find-comment.outputs.comment-id != '' }}
+    steps:
+      - name: Find "!test" Comment by Maintainers
+        id: find-comment
+        uses: peter-evans/find-comment@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          issue-number: ${{ github.event.pull_request.number }}
+          comment-author: |
+            joshzcold
+            karlromets
```

Comment:
Could someone in their branch update this list with their own user and then subsequently trigger the e2e tests?

Reply:
Took a look at the github docs and saw that these triggers are only for the "default branch"